-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 7, Failures: 2, Errors: 0, Skipped: 5, Time elapsed: 1.784 s <<< FAILURE! - in TestSuite
loginTrue(com.course.cases.LoginTest)  Time elapsed: 0.509 s  <<< FAILURE!
org.apache.http.conn.HttpHostConnectException: Connect to localhost:8888 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
	at com.course.cases.LoginTest.getResult(LoginTest.java:77)
	at com.course.cases.LoginTest.loginTrue(LoginTest.java:48)
Caused by: java.net.ConnectException: Connection refused: connect
	at com.course.cases.LoginTest.getResult(LoginTest.java:77)
	at com.course.cases.LoginTest.loginTrue(LoginTest.java:48)

loginFalse(com.course.cases.LoginTest)  Time elapsed: 0.059 s  <<< FAILURE!
org.apache.http.conn.HttpHostConnectException: Connect to localhost:8888 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
	at com.course.cases.LoginTest.getResult(LoginTest.java:77)
	at com.course.cases.LoginTest.loginFalse(LoginTest.java:61)
Caused by: java.net.ConnectException: Connection refused: connect
	at com.course.cases.LoginTest.getResult(LoginTest.java:77)
	at com.course.cases.LoginTest.loginFalse(LoginTest.java:61)

